### ISSUE TITLE
URLs dynamically generated in JavaScript

### ISSUE THREAD

**Issue by trusktr** [xroche/httrack#141]
Seems like there's some JS that is generating URLs, but it isn't statically analyzable. How can I get this to work? Maybe I can just add the URLs to the list of URLs in the command?

**Comment by mpheyse** [comment]
Yes, you can add as many base URLs as you want. And it is a good way to get around link paths that don't meet up.

**Comment by marspe** [comment]
What if the website has many pages and a lot of URLs generated by javascript? How to manage with the problem of archiving such URLs?

**Comment by xroche** [comment]
Links that a page generates in JavaScript can't be found by HTTrack, which doesn't execute JS. That's a known limitation rather than a bug. Closing. Sorry for the long delay.
